test/reflect.test.ts

```diff
@@ -210,8 +224,5 @@ test("computed object name", () => {
 
 test("err", () => {
   const fn = () => {};
-  const result = assertNodeKind<Err>(reflect(fn), NodeKind.Err);
-  expect(result.error.message).toEqual(
-    "Functionless reflection only supports function parameters with bodies, no signature only declarations or references. Found fn."
-  );
+  expect(reflect(fn)).toBeUndefined();
```

Comment:
Should this test continue to error?

Reply:
No, callers of reflect throw this error, not `reflect` - i.e. `Function`, `StepFunction`, etc. call call `validateFunctionLike`